tests/python/relax/test_vm_codegen_only.py

```diff
@@ -421,5 +421,89 @@ def main() -> R.Tensor((4,), dtype="float32"):
     tvm.testing.assert_allclose(res.numpy(), np.ones((4,), "float32"))
 
 
+@pytest.fixture
+def packed_func_check_if_exists():
+    func_name = "testing.check_if_none"
+
+    cached = tvm.get_global_func(func_name, allow_missing=True)
+
+    @tvm.register_func(func_name, override=True)
```

Comment:
move this registeration to relax.testing namespace

Reply:
Sounds good, and updated!